assets/translations/fr.json

```diff
@@ -4,5 +4,9 @@
     "homeEmergency": "Une URGENCE ?",
     "homeClickEmergency": "Appelez directement en cliquant\nsur ce bouton",
     "homeWaiting": "En attendant, vous\npouvez peut-être",
-    "homeWaitingAct": "AGIR"
+    "homeWaitingAct": "AGIR",
+    "aboutTitle": "L'association",
+    "aboutSubTitle1": "Qui sommes-nous ?",
+    "aboutPart1": "",
```

Comment:
no value ?

Reply:
will be filled in the next PR for the about page